cumulusci.yml

```diff
@@ -92,21 +91,22 @@ tasks:
             field: hed__Application_Type__c
             values: "Elementary School,Middle School,High School"
             existing_values: "Elementary School,Middle School,High School"
-            sorted: true
+            sorted: True
             namespaced: False
 
     add_behavior_response_type_values:
         description: Adds additional picklist values to the Type field.
-        class_path: tasks.add_picklist_values.AddPicklistValues
+        class_path: cumulusci.tasks.metadata_etl.picklists.AddPicklistEntries
         group: 'K-12: Metadata'
         options:
-            sobject: hed__Behavior_Response__c
-            field: hed__Type__c
-            values: "Detention,Expulsion with Services,In-School Suspension,Out-of-School Suspension,Removal from Classroom"
-            restricted: true
-            sorted: True
-            namespaced: False
-
+            picklists: hed__Behavior_Response__c.hed__Type__c
+            entries:
+                - fullName: "Award"
+                  label: "Award"
+                  add_before: "Expulsion"
+                - fullName: "Verbal Recognition"
+                  label: "Verbal Recognition"
+                  add_before: "Other"
```

Comment:
Please modify this to include the original picklist values in EDA as well above the new ones in the order of Other, Expulsion, and Academic Probation.